meta/boardreports/2020-10.md

```diff
@@ -0,0 +1,68 @@
+# InnerSource Patterns WG - Report for governance meeting 2020-10
+
+**Data from June to September**
+
+## Mission - Briefly describe what your project actually does.
+
+- Discuss community InnerSource practices and dynamics, collect and document agreed upon best practices of how to do InnerSource - in the form of patterns
+- Continuously publish the most mature patterns as an ebook and website
```

Comment:
I just want to point out that the mission statement here is slightly different from what we have. in our repo README. Just pasting the original text in verbatim here for reference. 

Doesn't matter much to me but maybe we should just stick to the original?

```suggestion
- Collect and document agreed upon best practices of how to do InnerSource - in the form of patterns
- Continuously publish the most mature patterns as an ebook
```